### PR TITLE
can view more than 200 events

### PR DESCRIPTION
**Pull request by cindytung**
derp

### CODE REVIEW

File: cal/cal/static/templates/tagDetails.html [hunk 1]
```diff
@@ -6,9 +6,13 @@ <h3 class="well">Average hours per {{$ctrl.timeStep}}: {{ $ctrl.averageHours }}<
...
     <nvd3 options="$ctrl.tagLine" data="$ctrl.ctrlGraphData"></nvd3>
-  </div>
-  <h3 class="well">Contained Events</h3>
-  <ul class="list-group" ng-repeat="tag in $ctrl.tagEvents">
-    <li class="list-group-item">{{ tag.start }}: {{ tag.name }}</li>
-  </ul>
+    <h3 class="well">Contained Events</h3>
+    <button ng-disabled="$ctrl.currentPage === $ctrl.lastPage - 1"
+      ng-click='$ctrl.currentPage=$ctrl.currentPage+1; $ctrl.showPageEvents();'>Previous</button>
+    <button ng-disabled="$ctrl.currentPage == 0"
```
Comment (jamesuejio) [inline cal/cal/static/templates/tagDetails.html:41]: OOO i like it

**Review by jamesuejio — APPROVED** [review]
Optimization for future to only load events when you click prev or next.

File: cal/cal/static/js/app.js [hunk 2]
```diff
@@ -223,9 +227,17 @@ function TagsDetailCtrl($scope, $interpolate, $http, QueryService) {
...
+    var start = this.currentPage * this.pageSize;
+    var end = (start + this.pageSize > this.tagEvents.length) ? this.tagEvents.length : start + this.pageSize;
+    this.pageEvents = [];
+    for (var i = start; i < end; i++) {
+      this.pageEvents.push(_this.tagEvents[i]);
+    }
+  }.bind(this);
+
+  this.getEvents = function(pageNum) {
+    $http({method: 'GET', url: tagUrl + '.json', params:{page:pageNum}}).
```
Comment (MaxFangX) [inline cal/cal/static/js/app.js]: You should install a linter - I'm getting yelled at here. Spacing should be `params: { page: pageNum }}`

File: cal/cal/static/js/app.js [hunk 3]
```diff
@@ -480,9 +509,17 @@ function CategoriesDetailCtrl($scope, $http, QueryService){
...
+    var start = this.currentPage * this.pageSize;
+    var end = (start + this.pageSize > this.categoryEvents.length) ? this.categoryEvents.length : start + this.pageSize;
+    this.pageEvents = [];
+    for (var i = start; i < end; i++) {
+      this.pageEvents.push(_this.categoryEvents[i]);
+    }
+  }.bind(this);
+
+  this.getEvents = function(pageNum) {
+    $http({method: 'GET', url: categoryUrl + '.json', params:{page:pageNum}}).
```
Comment (MaxFangX) [inline cal/cal/static/js/app.js]: same spacing comment. Check out `jslint` and `jshint` for your text editor

**Review by MaxFangX — APPROVED** [review]
+1

**Comment by MaxFangX** [comment]
+1 After implementing changes can merge. Good stufff

**Pull request merged**